components/html-block/README.md

```diff
@@ -1,44 +1,139 @@
+  import '@brightspace-ui/core/components/icons/icon.js';
 </script>
-
 <d2l-html-block>
-  <template>
-    <!-- some html -->
-  </template>
+	<template>
+		<!-- docs: start hidden content -->
+		<style>
+			div {
+				--d2l-icon-fill-color: var(--d2l-color-cinnabar);
+			}
+			span {
+				color: var(--d2l-color-cinnabar);
+				margin-left: 10px;
+				vertical-align: middle;
+			}
+			d2l-icon {
+				align-self: center;
+				flex-shrink: 0;
+			}
+			.warning-container {
+				align-items: center;
+				display: flex;
+				justify-content: center;
+			}
+		</style>
+<!-- docs: end hidden content --><div class="warning-container">
+			<d2l-icon icon="tier3:alert"></d2l-icon>
+			<span>
+				<b>Important:</b> user-authored HTML must be trusted or properly sanitized!
+			</span>
+		</div>
+	</template>
 </d2l-html-block>
 ```
 
-To use `d2l-html-block` within another Lit component, use the `unsafeHTML` directive to avoid escaping the HTML.
+To use `d2l-html-block` within another Lit component, use the [unsafeHTML](https://lit-html.polymer-project.org/guide/template-reference#unsafehtml) directive to avoid escaping the HTML.
 
-```javascript
-import { html, LitElement } from 'lit-element/lit-element.js';
-import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
+```html
+<script type="module">
+	import { html, LitElement } from 'lit-element/lit-element.js';
+	import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
+	import '@brightspace-ui/core/components/icons/icon.js';
 
-class SomeComponent extends LitElement {
+	class SomeComponent extends LitElement {
+		render() {
+			return html`
+				<d2l-html-block>
+					<template>${unsafeHTML(this._unsafeHTML)}</template>
+				</d2l-html-block>`;
+		}
+	}
 
-  render() {
-    return html`
-      <d2l-html-block>
-        <template>${unsafeHTML(this._someHTML)}</template>
-      </d2l-html-block>
-    `;
-  }
+	customElements.define('d2l-some-component', SomeComponent);
+</script>
+<d2l-html-component></d2l-html-component>
+```
 
-}
+### Rendering MathML and LaTeX
+Examples are provided to display how user-authored math can be embedded within your webpage.
 
-customElements.define('d2l-some-component', SomeComponent);
+**MathML:**
+<!-- docs: demo code -->
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/html-block/html-block.js';
+  import '@brightspace-ui/core/components/icons/icon.js';
+</script>
+<d2l-html-block>
+	<template>
+		<div class="mathml-container">
+			<math xmlns="http://www.w3.org/1998/Math/MathML">
+				<msqrt>
+					<mn>3</mn>
+					<mi>x</mi>
+					<mo>&#x2212;</mo>
+					<mn>1</mn>
+				</msqrt>
+				<mo>+</mo>
+				<mo stretchy="false">(</mo>
+				<mn>1</mn>
+				<mo>+</mo>
+				<mi>x</mi>
+				<msup>
+					<mo stretchy="false">)</mo>
+					<mn>2</mn>
+				</msup>
+			</math>
+		</div>
+	</template>
+</d2l-html-block>
 ```
 
+**LaTeX:** Rendering LaTeX requires the `us125413-mathjax-render-latex` feature flag to be enabled.
```

Comment:
Should we mention the exact feature flag here? There is some code that references the feature flag as well (which could be hidden)

Reply:
hmm yeah since the site is currently internal it's probably useful for now